main.go

```diff
@@ -75,14 +75,20 @@ func generate(name string, args []string) {
 		log.Fatalln("error parsing version:", err)
 	}
 
-	addExtRegexp, err := regexp.Compile(*addext)
-	if err != nil {
-		log.Fatalln("error parsing extension inclusion regexp:", err)
+	var addExtRegexp *regexp.Regexp = nil
+	if *addext != "" {
```

Comment:
I see, thanks.

I think it would help.

If I am a user, and I read `glow generate -help` and it said:

```
-addext string
    	Regular expression of extensions to include (e.g., .*)
```

I would expect `-addext 'foo'` to use whatever string I give it (including empty string) as the regular expression. Or, if I worry that empty regular expression is weird, I might have to look at the source and check what it actually does.

But if we just briefly mention that the regular expression is only used if it's not empty in the flag description, I think that'd be a nice thing that would remove the need for all of the above.

We can use go generate's `-run` flag description as an example:

```
Go generate accepts one specific flag:

	-run=""
		if non-empty, specifies a regular expression to select
		directives whose full original source text (excluding
		any trailing spaces and final newline) matches the
		expression.
```

So how about something like this:

```
-addext string
    	If non-empty, regular expression of extensions to include (e.g., .*)
```